common/lib/capa/capa/responsetypes.py

```diff
@@ -709,6 +709,12 @@ def setup_response(self):
         self.correct_choices = set([choice.get(
             'name') for choice in correct_xml])
 
+        incorrect_xml = self.xml.xpath('//*[@id=$id]//choice[@correct="false"]',
+                                     id=self.xml.get('id'))
+
+        self.incorrect_choices = set([choice.get(
+            'name') for choice in incorrect_xml])
```

Comment:
This line and the line above it are not broken in a readable way. Please see https://github.com/edx/edx-platform/wiki/Python-Guidelines particularly the section on "Breaking long lines"